src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../card";
+
+import * as styles from './more-blogs-section.module.scss';
+
+const MoreBlogsSection = ({ data, refMoreFrom, title }) => {
+  const { edges } = data.allMdx;
+  return (
+    <div className={styles.moreFromXlContainer} ref={refMoreFrom} >
+      <div className={styles.titleContainer}>
+        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>{`${title} →`}</Link>
+      </div>
+      <div className={styles.blogsContainer}>
+        {edges.map(({node}) => <Card data={node} key={node.id} className={styles.cardStyles} />)}
+      </div>
+    </div>
+  );
+}
+
+export { MoreBlogsSection };
+
+MoreBlogsSection.propTypes = {
+  refMoreFrom: PropTypes.object,
+  title: PropTypes.string,
```

Comment:
please add default values for these

Reply:
Done? I don't know if what I wrote is okay

Reply:
If some prop isn't required, you should add the default value for these props. In this case, title and refMoreFrom aren't required, so you should define their default values